Dockerfile

```diff
@@ -5,13 +5,13 @@
 # $ docker run -d --rm -p <local-port>:80 --name malibu-container malibu
 # 
 
-FROM php:7.3-apache-buster
+FROM docker.io/php:8.2-apache-bullseye
 
 ENV JQUERY 3.6.0
 ENV CLIPBOARD 2.0.8
 
-RUN apt-get update \
-    && apt-get install --no-install-recommends -y yaz libyaz-dev wget unzip python3-bs4 python3-requests \
+RUN apt-get update && apt-get upgrade -y
+RUN apt-get install --no-install-recommends -y yaz libyaz-dev wget unzip python3-bs4 python3-requests \
```

Comment:
It is recommened to "always combine RUN apt-get update with apt-get install in the same RUN statement", see https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#apt-get